_pages/ato.md

```diff
@@ -1,18 +1,77 @@
+
+* Not relevant to the system
+* Relevant to the system, and is one of the following:
+    * Taken care of by one of:
+        * The platform (cloud.gov)
+        * The framework (Django, etc)
+        * The authorization layer
+        * Etc.
+    * Implemented by you
+    * Not yet taken care of
+
+An "inherited" control would be something like FedRAMP requiring that fire extinguishers be present near the servers, which is something that AWS needs to worry about for _their_ compliance, and we don't need to re-explain when launching an application hosted there. Your Masonry file essentially contains the "overrides".
+
+The controls that are _not_ inherited from an underlying system must be listed in your Masonry file with a short explanation ("narrative"), and "implemented" before the system can receive an ATO. "Implementation" in the compliance sense is the same as in the code sense: ensure that the system meets that requirement, based on current industry best practices.
+
+### Step 4 — Assess the controls
+
+In other words, "verify that your system is secure". The first step in doing so is to run the [security scans](../security/scanning/). This is a preliminary assessment, final assessment will be done in collaboration with the GSA Office of the Chief Security Officer (OCISO).
+
+### Step 5 — Complete documentation package
+
+Fill out the documentation in the checklist. We hope to auto-create the SSP from the Compliance Masonry YAML file at some point, but for now you should copy and paste.
+
+The full list of data and functions in and of the system (in government parlance "mission based information types" and "delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the government has a formalized data set of mission functions that should be mapped to the system via [NIST 800-60](http://csrc.nist.gov/groups/SMA/fisma/categorization.html). For a Rails app, for example, this can simply be a link to the `db/schema.rb` file on GitHub.
+
+### Step 6 — Authorize the system
+
+Your Infrastructure Lead will work with you to schedule and prioritize your system assessment. Once assessment starts, the first step is that the Authorizing Official will review all the items in your ATO checklist including all the documents you generated. When the items in the ATO checklist are ready, your Infrastructure Lead will circulate the Rules of Engagement document for signatures.
+
+Then, a team with members from the project team, TTS Infrastructure, your Infrastructure Lead, and the GSA OCISO will convene for at least a week as the [ATO Sprinting Team](https://docs.google.com/document/d/1bGOV_pp_BlAzZsoa2D5pnsinx3R2gVnaBdFvHkwv0ig/edit), and begin to follow the [Lightweight Security Authorization Process](https://insite.gsa.gov/portal/content/627230).
```

Comment:
The Lightweight Security Authorization Process won't apply to _every_ ATO, right? Or is it just that LATO doesn't?